packages/manager/modules/pci/src/projects/project/storages/legacy/containers/containers.service.js

```diff
@@ -50,11 +50,10 @@ export default class PciStoragesContainersService {
   }
 
   getAccessAndToken(projectId) {
-    return this.OvhApiCloudProjectStorage.v6()
-      .access({
-        projectId,
-      })
-      .$promise.then(({ token, endpoints }) => ({
+    return this.$http
+      .post(`/cloud/project/${projectId}/storage/access`)
+      .then(({ data }) => data)
+      .then(({ token, endpoints }) => ({
```

Comment:
Same as https://github.com/ovh/manager/pull/9541#discussion_r1247956195

Reply:
it fixed